src/core/Akka.Tests/Actor/Stash/StashCapacitySpecs.cs

```diff
@@ -0,0 +1,145 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="StashCapacitySpecs.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2023 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2023 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Configuration;
+using Akka.TestKit;
+using Xunit;
+using Xunit.Abstractions;
+using FluentAssertions;
+
+namespace Akka.Tests.Actor.Stash;
+
+public class StashCapacitySpecs : AkkaSpec
+{
+    public StashCapacitySpecs(ITestOutputHelper output) : base(Config.Empty, output: output)
+    {
+        
+    }
+
+    [Fact]
+    public async Task ShouldGetAccurateStashReadingForUnboundedStash()
```

Comment:
Basic happy path stash spec. Going to add one for dealing with bounded stashes once I get the other half of #6658